.storybook/main.ts

```diff
@@ -2,12 +2,26 @@ import { StorybookConfig } from "@storybook/react-webpack5"
 import TsconfigPathsPlugin from "tsconfig-paths-webpack-plugin"
 import { exec as execCb } from "child_process"
 import { promisify } from "util"
+import * as fs from "fs"
+import * as path from "path"
 
 const exec = promisify(execCb)
 const getGitSha = async (): Promise<string> => {
   const { stdout } = await exec("git rev-parse HEAD")
   return stdout.trim()
 }
+// We have postinstall hooks disabled, so ensure that MSW is initialized
```

Comment:
this check needs to happen in `main.ts` not `preview.ts` (where storybook integrates with msw) because `main.ts` is the nodejs file; preview runs in browser. (Potentially `preview` could fetch the file, but this is much simpler.